app/google-services.json

```diff
@@ -0,0 +1,40 @@
+{
```

Comment:
To keep this PR simpler, can we move the JSON file to its own, dedicated PR that this one is blocked on? I feel like this PR is doing a couple different things. Keeping it separate will make the review a bit easier, and let us focus on the individual aspects of each PR.

Reply:
Okay so in terms of the security thing, the risk is that if we put up the config file our api key will be exposed. However, that API key can only be used to fiddle with our backend (if any) on our firebase console. 
Since we don't have any, it won't be an issue.  

The reason I am keen on merging it together is, without the config file the app won't build thereby creating a halt in everyone's work. Also it's a config of a test project, we'll have to change it whenever we decide to release. 

Other alternative - We can tell our developers to add their own config files as a part of setting up the project. The downside to this is, all of them will have to additionally go to Firebase, create a project, register the app and put the config file to be able to run it. It might be okay for the existing devs, but I don't think that will be the case for the new contributors.

Reply:
This has moved to another PR so I am resolving this